### PR TITLE
added testclass filter for testclasses and bundles

### PR DESCRIPTION
**Pull request by DianaSlepikaite**
Signed-off-by: Diana Slepikaite <d.slepikaite@gmail.com>

### CODE REVIEW

**Comment by DianaSlepikaite** [comment]
rerun tests

**Comment by DianaSlepikaite** [comment]
run tests

**Comment by Michael-Baylis** [comment]
rerun tests

File: src/app/results-page/rasfilters/testclasses-filter/testclasses-filter.component.ts [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+            var newTestclasses : Object[]=[];
+            var newBundles: Object []=[];
+            var nextId = 0;
+            for (let testClass of result.testclasses) {
+              newTestclasses.push({content:testClass.testclass, id:nextId});
+              // whitout id so it woudnt be unique elements
+              newBundles.push(testClass.bundle);
+              nextId++;
+            }
+            var nextId = 0;
```
Comment (Michael-Baylis) [inline src/app/results-page/rasfilters/testclasses-filter/testclasses-filter.component.ts]: Dont need to do anything with Bundles,  you can remove that code.

Comment (Michael-Baylis) [reply src/app/results-page/rasfilters/testclasses-filter/testclasses-filter.component.ts]: Bundles will have its own dropdown

**Review by Michael-Baylis — CHANGES_REQUESTED** [review]

**Review by Michael-Baylis — APPROVED** [review]

**Pull request merged**